### PR TITLE
[T4-automation]:CEPH-83575470 - URI query parameters values in the RGW ops log file and rgw log details

### PR DESCRIPTION
**Pull request by anrao19**
URI query parameters values in the RGW ops log file : 
 log: http://magna002.ceph.redhat.com/cephci-jenkins/Anuchaithra/test_rgw_opslog.console.log

RGW log details:
http://magna002.ceph.redhat.com/cephci-jenkins/Anuchaithra/test_rgw_log_details.console.log

### CODE REVIEW

File: rgw/v2/tests/s3cmd/configs/test_rgw_log_details.yaml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# script: test_rgw_ops_log.py
+# Bugs: 2188557
+# Polarian: CEPH-83575470
+config:
+  user_count: 1
+  bucket_count: 2
+  objects_count: 100
+  test_ops:
+    rgw_log_detail: true
```
Comment (hmaheswa) [inline rgw/v2/tests/s3cmd/configs/test_rgw_log_details.yaml:9]: @anrao19 , are we intentionally not using `rgw_ops_log: true` here
I dont see any of s3 ops like upload object etc.. in the log
http://magna002.ceph.redhat.com/cephci-jenkins/Anuchaithra/test_rgw_log_details.console.log

Comment (anrao19) [reply rgw/v2/tests/s3cmd/configs/test_rgw_log_details.yaml:9]: test_ops is do just for the if condition , not set anything.
Here we are just getting rgw log details, if log_to_file not set we will set then we will get the details

File: rgw/v2/tests/s3cmd/test_rgw_ops_log.py [hunk 2]
```diff
@@ -113,12 +126,79 @@ def test_exec(config, ssh_con):
...
+
+    if config.test_ops.get("rgw_log_detail", False):
+        log.info("Verify rgw logs exist and rgw log details")
+        ceph_version_id, _ = utils.get_ceph_version()
+        rgw_log_dict = {}
+        rgw_log_dict[ceph_version_id] = {}
+        root_path = str(Path.home()) + "/rgw_log_details.yaml"
+        log.info(f"Verify file exist {root_path}, if not create file")
+        if not os.path.exists(root_path):
+            utils.exec_shell_cmd(f"touch {root_path}")
```
Comment (hmaheswa) [inline rgw/v2/tests/s3cmd/test_rgw_ops_log.py:169]: @anrao19 
as an enhancement in the coming PR's, you could add code in the else part here to compare the previous log size with the current log size of every rgw daemon on every host (if greater than one GB)

we can get only the size of a file using this
```
[root@ceph-pri-hsm-ms-izl3ix-node5 ~]# stat -c %s /var/log/ceph/0c1b712c-391b-11ef-a58e-fa163e71c208/ceph-client.rgw.shared.pri.ceph-pri-hsm-ms-izl3ix-node5.hegihi.log
24743703498
[root@ceph-pri-hsm-ms-izl3ix-node5 ~]#```

**Review by hmaheswa — APPROVED** [review]

File: rgw/v2/tests/s3cmd/test_rgw_ops_log.py (hunk 2 above)
Comment (anrao19) [reply rgw/v2/tests/s3cmd/test_rgw_ops_log.py:169]: Sure @hmaheswa

**Review by ckulal — COMMENTED** [review]
LGTM

**Review by hmaheswa — APPROVED** [review]

**Review by viduship — APPROVED** [review]

**Review by ckulal — APPROVED** [review]

**Pull request merged**